app/src/main/java/com/simply/birthdayapp/presentation/ui/components/BaseTextField.kt

```diff
@@ -0,0 +1,64 @@
+package com.simply.birthdayapp.presentation.ui.components
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.simply.birthdayapp.presentation.ui.theme.BackgroundColor
+import com.simply.birthdayapp.presentation.ui.theme.Primary2
+
+@Composable
+fun BaseTextField(
+    textState: String,
+    label: String = "",
+    onValueChange: (String) -> Unit = {},
+    keyboardType: KeyboardType = KeyboardType.Text,
+    shape: Shape = TextFieldDefaults.shape,
```

Comment:
There's a practice when we pass the lambda function arguments as the last one, so we can have the ability to open closures when we call this composable. This should work too if we use named arguments, but as a good practice consider it that way.

Reply:
fixed